compiler/rustc_parse_format/src/lib.rs

```diff
@@ -9,13 +9,10 @@
     html_playground_url = "https://play.rust-lang.org/",
     test(attr(deny(warnings)))
 )]
-#![cfg_attr(not(bootstrap), doc(rust_logo))]
-#![cfg_attr(not(bootstrap), allow(internal_features))]
-#![cfg_attr(not(bootstrap), feature(rustdoc_internals))]
 #![deny(rustc::untranslatable_diagnostic)]
 #![deny(rustc::diagnostic_outside_of_impl)]
-// We want to be able to build this crate with a stable compiler, so no
-// `#![feature]` attributes should be added.
+// WARNING: We want to be able to build this crate with a stable compiler,
+//          so no `#![feature]` attributes should be added!
```

Comment:
Someday I might open an MCP for `#![deny(rustc::features)]` or a lang FCP for `#![deny(features)]` :^)

Reply:
I think a simple PR to add the lint would be enough for that. Lints are explicitly not stable, and this is a lint for an *unstable* feature (the feature attribute).